src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,76 @@
+package strawman
+package collection
+package mutable
+
+trait Seq[A] extends strawman.collection.Seq[A] with Growable[A] {
```

Comment:
No, not intentional. I am not sure yet what the right scehem should be. It seems natural that mutable sequences should not just have `update`, but also `insert`, `remove`, `+=`. But then what to do about `WrappedArray`? One possibility is to also have these things for `WrappedArray` and accept that e.g. `+=` has linear cost. That's what current collections seem to do. Another possibility would be to back `WrappedArray` by a `ResizableArray`.